### ISSUE TITLE
Show load info when hovering plasmoid

### ISSUE THREAD

**Issue by pozemka** [rbn42/pixelload#5]
It would be nice to show some info when hovering plasmoid with cursor. Such as CPU% load (or few most consuming applications) RAM/Swap (total, used, free, cached) usage info, IO and Network statistic. 
See built in System monitor plasmoid for example.

**Comment by rbn42** [comment]
Yes, that is a good idea. But sorry, I don’t have time to add new features to this widget.

**Comment by rbn42** [comment]
BTW, I will be glad to transfer this repo to anyone who wants to contribute.

**Comment by pozemka** [comment]
> Yes, that is a good idea. But sorry, I don’t have time to add new features to this widget.

I see.
Anyway thank you for this widget. I will continue to use it
